.taskcluster.yml

```diff
@@ -124,7 +124,6 @@ tasks:
             - "/bin/bash"
             - "-lcx"
             - "apt-get -qq update &&
```

Comment:
Looks like we can remove the update too.

Reply:
Right! Done in fe09616

Reply:
Actually, in c5d92ae.